CMakeLists.txt

```diff
@@ -45,6 +45,11 @@ option(SNAPPY_BUILD_TESTS "Build Snappy's own tests." ON)
 
 option(SNAPPY_FUZZING_BUILD "Build Snappy for fuzzing." OFF)
 
+option(SNAPPY_OSSFUZZ_BUILD "Build Snappy for oss-fuzz." OFF)
```

Comment:
`SNAPPY_OSSFUZZ_BUILD` is set by the oss-fuzz builder. See https://github.com/google/oss-fuzz/pull/2874/files#diff-92672b1afbd2ebe31528dff0bd201f41R22-R23